jdaviz/configs/default/plugins/model_fitting/tests/test_fitting.py

```diff
@@ -171,8 +170,9 @@ def test_cube_fitting_backend():
     # Check that spectrum result is formatted as expected.
     assert type(fitted_spectrum) == Spectrum1D
     assert len(fitted_spectrum.shape) == 3
-    assert fitted_spectrum.shape == (IMAGE_SIZE, IMAGE_SIZE, SPECTRUM_SIZE)
+    assert fitted_spectrum.shape == (IMAGE_SIZE_X, IMAGE_SIZE_Y, SPECTRUM_SIZE)
     assert fitted_spectrum.flux.unit == u.Jy
+    assert not np.all(fitted_spectrum.flux.value == 0)
```

Comment:
I don't think it would be too hard to generate a known shape with some noise on top and check that the result is within tolerance of the expected fit, but I agree that I think we can defer that for now.

Reply:
There's a comment to that effect on line 149 `# n_cpu = 1  # NOTE: UNCOMMENT TO DEBUG LOCALLY, AS NEEDED`, think there needs to be another one here?

Reply:
Instead of copy pasting that sentence everywhere in the test, should I just write it up in dev docs? Any suggestion on where?

Reply:
this is currently the one place that will catch these failures, right?  But I guess we may have more in the future - I'm just trying to save us time in the future if/when this returns zeros and trips this test again.  I'm not sure I would remember and see the other comment to even know that a debug mode is an option.  But I suppose we always have the blame history back to this PR 🤞

Reply:
I will ping you to review #1346 when it is ready. Thanks!